v3/plugins/scalameta/metals/1.7.2/meta.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v2
+publisher: scalameta
+name: metals
+version: 1.7.2
+type: VS Code extension
+displayName: Scala (Metals)
+title: Scala language server with rich IDE features
+description: This plug-in provides support for Scala development via the Metals language server.
+icon: https://rawcdn.githack.com/OlegIlyenko/scala-icon/c453c6f67dfcc1dbf0ca1e437d32d04f1d710983/scala-icon.svg
+repository: https://github.com/scalameta/metals-vscode
+category: Language
+firstPublicationDate: "2020-01-24"
+spec:
+  containers:
+    - image: "quay.io/eclipse/che-sidecar-java:11-f76ca45"
+      name: vscode-metals
+      memoryLimit: "1500Mi"
+      volumes:
+      - name: sbt
+        mountPath: /home/theia/.sbt
+      - name: ivy2
+        mountPath: /home/theia/.ivy2
+      - name: cache
+        mountPath: /home/theia/.cache
```

Comment:
See my comment elsewhere, I think a better approach is to have volumes for `.cache/metals` and `.cache/coursier` if they need to be persisted, since linking the `.cache` directory in general to a PVC could cause problems.

Reply:
so what am I supposed to do, since `mountPath: /home/theia/.cache/coursier` didn't work?

Reply:
Sorry, I'm not being very clear. Mounting `mountPath: /home/theia/.cache/coursier` worked for coursier, but failed because we couldn't create `/home/theia/.cache/metals`. It should work without problem if we have 
```yaml
      - name: coursier
        mountPath: /home/theia/.cache/coursier 
      - name: metals
        mountPath: /home/theia/.cache/metals 
```

Reply:
done